crates/ruma-common/Cargo.toml

```diff
@@ -42,6 +42,7 @@ unstable-msc3551 = ["unstable-msc1767"]
 unstable-msc3552 = ["unstable-msc3551"]
 unstable-msc3553 = ["unstable-msc3552"]
 unstable-msc3554 = ["unstable-msc1767"]
+unstable-msc3783 = []
```

Comment:
Can you re-export this feature from `ruma`?

Reply:
Should I add it to the CI feature as well since we have a feature specific test?

Reply:
Yes, please :)